thaw/src/table/table.css

```diff
@@ -1,6 +1,6 @@
 .thaw-table {
     display: table;
-    table-layout: fixed;
+    table-layout: auto;
```

Comment:
This was breaking the tables  (text was overlapping).

Reply:
We'd better do the same with FluentUI here.

We have several other ways to solve this problem, for example:

1. [Resizable Columns](https://react.fluentui.dev/iframe.html?viewMode=docs&id=components-table--default#resizable-columns--preview-)
2. Set the style for the Table component. [Take a look under the `Enhanced attribute spreading` heading](https://github.com/leptos-rs/leptos/releases/tag/0.7.0-alpha)